src/open_company_web/components/topic.cljs

```diff
@@ -61,64 +64,61 @@
                               utils/oc-animation-duration)
               growth-fade (Fade.
                             growth-children
-                            (if expanded 0 1)
-                            (if expanded 1 0)
+                            (if expanded? 0 1)
+                            (if expanded? 1 0)
                             utils/oc-animation-duration)]
             (.play growth-resize)
             (.play growth-fade)))
       ;; show/hide topic more ellipses
       (.play
         (Fade.
           topic-more
-          (if expanded 0 1)
-          (if expanded 1 0)
+          (if expanded? 0 1)
+          (if expanded? 1 0)
           utils/oc-animation-duration))
       ;; expand/collapse the ellipses div
       (.play
         (Resize.
           topic-more
-          #js [body-width (if expanded 0 20)]
-          #js [body-width (if expanded 20 0)]
+          #js [body-width (if expanded? 0 20)]
+          #js [body-width (if expanded? 20 0)]
           utils/oc-animation-duration))
       ;; show/hide data and author div
       (.play
         (Fade.
           topic-date
-          (if expanded 1 0)
-          (if expanded 0 1)
+          (if expanded? 1 0)
+          (if expanded? 0 1)
           utils/oc-animation-duration))
       ;; animate body and navigation div height
       (let [height-animation (Resize.
                                body-nav-node
-                               #js [body-width (if expanded (+ body-height 20) 0)]
-                               #js [body-width (if expanded 0 (+ body-height 20))]
+                               #js [body-width (if expanded? (+ body-height 20) 0)]
+                               #js [body-width (if expanded? 0 (+ body-height 20))]
                                (* 0.8 utils/oc-animation-duration))]
         (doto height-animation
           (events/listen
            EventType/FINISH
            (fn [e]
-            (om/update-state! owner :expanded not)
-            (.setTimeout js/window
-              #(setStyle body-nav-node #js {:overflow (if expanded "hidden" "visible")
-                                            :height (if expanded "0" "auto")}) 1)))
+             (when-not expanded? (.setTimeout js/window #(scroll-to-topic-top topic) 100))))
           (.play)))
 
-      (scroll-to-topic-top topic)
 
-      (if-not expanded
+      (if-not expanded?
         ;; show the edit button if the topic body is empty
         (let [section (keyword (:section-name options))
               section-data (:section-data data)
               body (utils/get-topic-body section-data section)]
-          (when (clojure.string/blank? body)
+          (when (and (clojure.string/blank? body) (:force-edit-cb options))
             ((:force-edit-cb options) true)))
         ;; hide the edit button if necessary
-        ((:force-edit-cb options) false)))))
+        (when-let [f (:force-edit-cb options)]
+          (f false))))))
```

Comment:
@martinklepsch yep, that was used for mobile editing. I probably missed it when i removed it. good catch